primitives/runtime-interface/test-wasm/src/lib.rs

```diff
@@ -123,6 +123,15 @@ pub trait TestApi {
 		data == 42
```

Comment:
weird indent (may be github).

Reply:
Yeah that is github, it is fucking it up for me as well.